website/docs/gen-ai/inference/Mistral-7b-inf2.md

```diff
@@ -231,6 +232,164 @@ Below screenshots provide some examples of the model response based on different
 
 ![Gradio Convo 2](img/mistral-conv-2.png)
 
+
+### Ray Head Node High Availability With Elastic Cache for Redis
+
```

Comment:
:::info

This Mistral7b deployment is using Ray Serve with High availability. If you have already deployed mistral7b in the previous steps, then you can delete the deployment and run the below steps.

:::